dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/dialogEditBindDomain.xml

```diff
@@ -93,6 +93,6 @@
         <id>domain.dnsserver</id>
         <label>DNS Server</label>
         <type>text</type>
-        <help>Set the DNS server hosting this file. This should usually be the FQDN of your firewall where the BIND plugin is installed.</help>
+        <help>Set the DNS server hosting this file. This should usually be the FQDN of your firewall where the BIND plugin is installed. In a real world setup, dont forget to add the required Resource Records.</help>
```

Comment:
```suggestion
        <help>Set the DNS server hosting this file. This should usually be the FQDN of your firewall where the BIND plugin is installed. Required Resource Records must be added separately.</help>
```

Reply:
What is meant here anyway? That the upstream DNS server needs to have the NS records of the BIND instance? For me just adding the additional text seems unclear.

Reply:
`@    NS nameserver.fqdn.`

is filled with the value the user is setting on the interface

what will not be added with the template of opnsense?
right the required(if NS is within the same zone) A/AAAA Resource Record.
[maybe i overlooked something - idk. correct me if i am wrong or the situation has changed without me noticing]
  hence the comment `don't forget....`


the basic requirements of a proper DNS setup are not touched with my 'hint'.
meaning: anyone setting up a valid globaly reachable domain&dns server, has to know anyway that the Registrar needs some Information for a working glue record.